Tests/bootstrap.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Bootstrap file for running the tests on PHPCS 3.x.
+ *
+ * Load the PHPCS autoloader and the WPCS PHPCS cross-version helpers.
+ *
+ * {@internal We need to load the PHPCS autoloader first, so as to allow their
```

Comment:
This comment is not needed in the context of the YoastCS standard as YoastCS does not need the aliasing as such.
It could be changed to refer to the fact that WPCS at this point in time still needs the aliases and that as you extend a WPCS class, it is needed here too.

As a side-note: WPCS will be dropping PHPCS 3.x support by the end of the summer, so at that time, the aliases will be removed from WPCS completely.